app/controllers/orders_controller.rb

```diff
@@ -1,43 +1,43 @@
 class OrdersController < ApplicationController
+  before_action :validate_user
+
+  def validate_user
+    if current_user.nil?
+      flash[:login] = {color: 'red', message: "Please log in to check out"}
+      redirect_to '/login'
+    end
+  end
 
   def index
     @orders = current_user.orders
   end
 
   def create
-    if current_user.nil?
-      flash[:login] = {color: 'red', message: "Please log in to check out"}
-      redirect_to '/login'
-    else
-      @order = current_user.orders.create(
-        status: "ordered",
-        total_price: total_price
-      )
-      add_items_to_order
-      flash[:order_success] = {color: 'green', message: "Order was successfully placed"}
-      redirect_to orders_path
-      session[:cart] = {}
-    end
+    @order = current_user.orders.create(
+      status: "ordered",
+      total_price: @cart.total_price
+    )
+    add_items_to_order
+    flash[:order_success] = {color: 'green', message: "Order was successfully placed"}
```

Comment:
Line is too long. [86/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.